src/view/xwayland.cpp

```diff
@@ -326,6 +327,15 @@ class wayfire_xwayland_view : public wayfire_xwayland_view_base
 
         on_request_move.set_callback([&] (void*) { move_request(); });
         on_request_resize.set_callback([&] (void*) { resize_request(); });
+        on_request_activate.set_callback([&] (void*)
+        {
+            if (!this->activated) {
```

Comment:
not sure if this will fix it, but `{` should be on the next line anyway.

Reply:
yes, that was the problem it seems